1.6.1-incubating/scala2.12-java8-ubuntu/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM eclipse-temurin:8-jre
```

Comment:
this docker file is using binary to build image

Reply:
We could make it configurable as the way in Spark's official dockerfile.
1. jre by default
2. set from arg `java_image_tag`

https://github.com/apache/spark/blob/0db63df2b2829f1358fb711cd657a22b7838ece2/resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile#L17
```
ARG java_image_tag=17-jre

FROM eclipse-temurin:${java_image_tag}
```

Reply:
@bowenliang123 it's not the Spark dockerfile.. see https://github.com/apache/spark-docker/blob/master/3.3.1/scala2.12-java11-ubuntu/Dockerfile

Reply:
Well noticed. While we could still consider using the configuarble tag in this case. Or just use JDK for more general usages.

Reply:
what do you think ? @cfmcgrady  @turboFei @yaooqinn   jre or jdk ?